sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationOperation.cs

```diff
@@ -229,7 +229,7 @@ async ValueTask<OperationState<AsyncPageable<DocumentStatusResult>>> IOperation<
             }
             else if (update.Value.Status == DocumentTranslationStatus.ValidationFailed)
             {
-                RequestFailedException requestFailedException = _diagnostics.CreateRequestFailedException(rawResponse, update.Value.Error, CreateAdditionalInformation(update.Value.Error));
```

Comment:
I think this is not necessary as it is doing what the RFE constructor will do anyways with the response error. The additional info part is also not needed as it is nesting "AdditionalInfo" under the "AdditionalInfo" section which seems wrong.